examples/plot_pvlaues.py

```diff
@@ -0,0 +1,193 @@
+# create a dictionary containing participants data
+limo_epochs = {str(subj): limo.load_data(subject=subj) for subj in subjects}
+
+# get key from subjects dict for easy slicing
+# subjects = list(limo_epochs.keys())
+
+# interpolate missing channels
+for subject in limo_epochs.values():
+    subject.interpolate_bads(reset_bads=True)
+
+# pick channels that should be included in the analysis
+picks_eeg = pick_types(limo_epochs['2'].info, eeg=True)
+# channels to be excluded
+exclude = ['EXG1', 'EXG2', 'EXG3', 'EXG4']
+
+# save epochs information (needed for creating a homologous
+# epochs object containing linear regression result)
+epochs_info = limo_epochs['2'].copy().drop_channels(exclude).info
+tmin = limo_epochs['2'].tmin
+
+###############################################################################
+# use epochs metadata to create design matrix for linear regression analyses
+
+# add intercept
+design = limo_epochs['2'].metadata.copy().assign(intercept=1)
+# effect code contrast for categorical variable (i.e., condition a vs. b)
+design['face a - face b'] = np.where(design['face'] == 'A', 1, -1)
+# create design matrix with named predictors
+predictors = ['intercept', 'face a - face b', 'phase-coherence']
+design = design[predictors]
+
+###############################################################################
+# --- run linear regression analysis using scikit-learn ---
+# data to be analysed
+data = limo_epochs['2'].get_data(picks_eeg)
+
+# number of epochs in data set
+n_epochs = data.shape[0]
+
+# number of channels and number of time points in each epoch
+# we'll use this information later to bring the results of the
+# the linear regression algorithm into an eeg-like format
+# (i.e., channels x times points)
+n_channels = picks_eeg.shape[0]
+n_times = len(limo_epochs['2'].times)
+
+# number of trials and number of predictors
+n_trials, n_predictors = design.shape
+# degrees of freedom
+dfs = float(n_trials - n_predictors)
+
+# vectorize (channel) data for linear regression
+Y = Vectorizer().fit_transform(data)
+
+# set up model and fit linear model
+linear_model = LinearRegression(fit_intercept=False)
+linear_model.fit(design, Y)
+
+###############################################################################
+# extract the coefficients for linear model estimator
+betas = get_coef(linear_model, 'coef_')
+
+# compute predictions
+predictions = linear_model.predict(design)
+
+# compute sum of squared residuals and mean squared error
+residuals = (Y - predictions)
+# sum of squared residuals
+ssr = np.sum(residuals ** 2, axis=0)
+# mean squared error
+sqrt_mse = np.sqrt(ssr / dfs)
+
+# raw error terms from each predictor in design matrix:
+# here, we take the inverse of a design) matrix projections
+# (i.e., A^T*A)^-1 and extract the square root of the diagonal values.
+error_terms = \
+    np.sqrt(
+        np.diag(
+            np.linalg.pinv(
+                np.dot(design.T, design)
+            )
+        )
+    )
```

Comment:
Can we use less then 5 lines for this ? ^^

Reply:
alright :)